docs/Actions.md

```diff
@@ -89,6 +89,44 @@ Use `gym --help` to get all available options.
 
 The alternative to `gym` is [`ipa`](#ipa) which uses [shenzhen](https://github.com/nomad/shenzhen) under the hood.
 
+### gradle
+
+You can use the `gradle` action to integrate your gradle tasks into `fastlane`.
+
+Simple usage:
+```ruby
+gradle(
+  task: 'assemble',
+  flavour: 'WorldDomination',
```

Comment:
Typo: flavour should be flavor.

I don't have permissions to fix this, can you fix this @lmirosevic  please?